Makefile

```diff
@@ -0,0 +1,17 @@
+PREFIX ?= /usr/local
+MAN_PREFIX ?= ${PREFIX}/man
+
+all:
+	cargo build --release
+
+install:
+	cargo build --release
+	install -d ${PREFIX}/bin
+	install -c -m 555 target/release/snare ${PREFIX}/bin/snare
+	install -d ${MAN_PREFIX}/man/man1
+	install -d ${MAN_PREFIX}/man/man5
+	install -c -m 444 snare.1 ${MAN_PREFIX}/man/man1/snare.1
+	install -c -m 444 snare.conf.5 ${MAN_PREFIX}/man/man5/snare.conf.5
+	install -d ${PREFIX}/share/examples/snare
+	install -c -m 444 snare.conf.example ${PREFIX}/share/examples/snare
+
```

Comment:
Was this trailing whitespace intentional?

Reply:
Yes, I should have put it in there at first. Old-fashioned Unix files are supposed to have a blank last line. I normally ignore this, but since this file is for old-fashioned Unix purposes, it seemed appropriate.

Reply:
OK, I didn't realise github rendered them. Or at least, usually it puts a funny symbol to show the absence of a trailing newline.

Anyway...